Sources/PaymentButtons/PayPalPayLaterButton.swift

```diff
@@ -27,7 +27,7 @@ public final class PayPalPayLaterButton: PaymentButton {
         insets: NSDirectionalEdgeInsets? = nil,
         color: Color = .gold,
         edges: PaymentButtonEdges = .softEdges,
```

Comment:
I notice some of the default sizes are left as `softEdges` while others were updated to `rounded`. Should all of the button inits use the same default, or is this expected to have them varied?

Reply:
That's true, we should have better consistency. I've changed them all to `.softEdges`